systemverilog-plugin/UhdmAst.cc

```diff
@@ -3352,6 +3357,32 @@ void UhdmAst::process_int_typespec()
     current_node->is_signed = true;
 }
 
+void UhdmAst::process_shortint_typespec()
+{
+    std::vector<AST::AstNode *> packed_ranges;   // comes before wire name
+    std::vector<AST::AstNode *> unpacked_ranges; // comes after wire name
+    current_node = make_ast_node(AST::AST_WIRE);
+    auto left_const = AST::AstNode::mkconst_int(16, true);
+    auto right_const = AST::AstNode::mkconst_int(0, true);
+    auto range = new AST::AstNode(AST::AST_RANGE, left_const, right_const);
+    packed_ranges.push_back(range);
```

Comment:
Please replace with ``make_range(16, 0)``: https://github.com/chipsalliance/yosys-f4pga-plugins/blob/main/systemverilog-plugin/UhdmAst.cc#L96

Reply:
Replaced, also changed this in another typespec handler.